examples/package.json

```diff
@@ -8,6 +8,6 @@
   },
   "devDependencies": {
     "jest": "^27.2.5",
-    "puppeteer": "^10.4.0"
+    "puppeteer": "^18.0.4"
```

Comment:
Same here. I don't know if the example still works, maybe you can test it?

Reply:
the same answer here WDYT?